### DISCUSSION TITLE
Recipe for fresh install?

### DISCUSSION THREAD

**Discussion by sageworksstudio** [ledermann/pingcrm#1360] [General]
This is really great to see Inertia/Rails7/Vue3 working together. You've done a nice job and it's a great showcase for this tech stack.

I don't mean to be "that guy," but do you by chance have a 'recipe' for starting from scratch? I.e., not the whole kitchen sink? Because there is so much going on, it's hard to tell how to start from scratch and the Inertia docs don't quite work the way this repo works.

**Comment by ledermann** [comment]
Thanks for your interest! Please take a look at my [Templatus](https://github.com/templatus) project, which is a starter kit for Rails and comes in several flavors. 

There is one for Inertia:
https://github.com/templatus/templatus-inertia

**Reply by sageworksstudio** [reply]
Wow, those are great, nice work! This is the second time Svelte has come across my radar this week, maybe it's time to give it a try.

Thanks!
